### ISSUE TITLE
Pearl Harbor doesn't like OpenCL

### ISSUE THREAD

**Issue by mathuin** [mathuin/TopoMC#17]
For some reason, the Pearl Harbor region does not work well with the OpenCL code for PrepRegion.  The inverse distance tree code works.

**Comment by mathuin** [comment]
Specific coordinates:  --ymax 21.3951 --xmin -158.0337 --ymin 21.2977 --xmax -157.9230

**Comment by mathuin** [comment]
Turns out the IDT tree fails silently by generating empty ocean!  Wacky.

So, the problem also exists for Alaska (well, Point Barrow anyway).  I think I've narrowed down the problem.

The conterminous US uses L01 for landcover values.  Puerto Rico uses L10.  Hawaii uses L07.  Alaska uses L04.  Both Hawaii and Alaska maps return maps in WGS 84 projection, whereas the rest use NAD 83.  Additionally, Hawaii and Alaska both have different Albers parameters than the conterminous US and Puerto Rico:  the latter have parallels of 29.5 and 45.5 and a center of 23,-96 while the former have parallels of 8 and 18 and a center of 3,-157 for Hawaii and parallels of 55 and 56 and a center of 50,-154 for Alaska.

I _think_ I need to use gdalwarp to transform all L04 and L07 files to match L01 parameters, before going any further.
